blog/2024-05-01-gsoc-intern.md

```diff
@@ -0,0 +1,12 @@
+---
+author: Antonin Delpeuch
+title: "Zyad joins us as a Google Summer of Code contributor"
```

Comment:
```suggestion
title: " Welcome to Zyad our 2024 Google Summer of Code intern"
```

Reply:
adding the year in the title for easy reference later on. Follow the pattern started here https://openrefine.org/blog/2020/05/04/outreachy-gsoc-projects

Reply:
"Welcome to Zyad our 2024 Google Summer of Code intern" feels slightly un-grammatical, I would expect a comma after Zyad at least

Reply:
Is the space before Welcome a typo?